### DISCUSSION TITLE
FLOSS: temas e datas

### DISCUSSION THREAD

**Discussion by fbarth** [Insper/open-dev#1285] [General]
Defina o tema, data e equipe (no máximo 3 pessoas) nesta thread.

**Comment by devfernandoa** [comment]
Linus Torvalds and Linux
Grupo: Fernando Alzueta, João Antonio Gomes, Luis Antonio Santana
Dia: 11/11

**Comment by RafaelMALima** [comment]
Richard Stallman and the FSF
Group: André Santos, Rafael Lima, Rafael Leventhal
27/11

**Comment by GuilhermeRameh** [comment]
Tema: RedHat
Grupo: Guilherme Rameh, Ilana Finger, Ananda Campelo
Dia: 13/11

**Comment by RicardoRibeiroRodrigues** [comment]
Canonical and Ubuntu
Group: Ricardo R. R., Guilherme Fontana, Thiago Rocha
Day: 27/11

**Comment by LidiaDomingos** [comment]
Mozilla
Group: Lidia Alves, Lincoln Rodrigo
Day: 18/11

**Comment by MekhyW** [comment]
The Debian project
Group: Felipe Catapano, Arthur Barreto, Lucas Hix

**Reply by fbarth** [reply]
Qual a data?

**Comment by Paulofalcao2002** [comment]
Eric Raymond (Cathedral and Bazaar)
Group: Leonardo Scarlato, Paulo Falcão e Rodrigo Mattar

**Reply by fbarth** [reply]
Qual a data?

**Comment by EricPossato** [comment]
KDE Community
Group: Eric Possato, Pedro Barão, Rodrigo Anciães
Date: 18/11

**Comment by G3mha** [comment]
Problemas de licenças Open Source
Group: Enricco Gemha, André Brito, Antonio Martins
Day: 13/11/2024

**Comment by yuritaba** [comment]
Larry Wall and Perl
Dia 11/11
Grupo: Yuri Tabacof, Rafael Paves, André Melo
